introduction.md

```diff
@@ -1,2 +1,9 @@
 I'm introducing myself to my dojo friends.
 My favorite CodeSchool path is Ruby.
+My deadly skills
+* HTML
+* CSS
+* Git
+* Ruby
+* PHP
+* C
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?